javascript/modules/client/src/client-addressList.ts

```diff
@@ -348,10 +351,13 @@ export class ClientAddressList extends ClientCore
     );
   }
 
-  private _buildUpdatePluginSettingsAction(params: IPluginSettings): DaoAction {
+  private _buildUpdatePluginSettingsAction(
+    pluginAddress: string,
+    params: IPluginSettings,
+  ): DaoAction {
     // TODO: check if to and value are correct
     return {
-      to: AddressZero,
+      to: pluginAddress,
```

Comment:
Could we do some quick check to make sure that the given string is an address? Ethers may have some helper for that